src/openloco/interop/hooks.cpp

```diff
@@ -77,6 +77,13 @@ static bool STDCALL fn_4054b9()
     return true;
 }
 
+FORCE_ALIGN_ARG_POINTER
```

Comment:
Why is it needed when there's no argument?

Reply:
Because syscalls like printf require a 16-byte aligned stack